.gitignore

```diff
@@ -55,3 +55,6 @@ docs/_site
 docs/Gemfile.lock
 /.gradle/
 /build/
+
+.metadata/
```

Comment:
I think that it's not necessary to change the .gitignore file. I will take it out from the PR. In my case, I had to change it because my eclipse workspace and the git repo are in the same directory and so eclipse created derived files that I didn't want to track.